lib/matplotlib/dates.py

```diff
@@ -274,8 +274,7 @@ def _from_ordinalf(x, tz=None):
 class strpdate2num(object):
     """
     Use this class to parse date strings to matplotlib datenums when
-    you know the date format string of the date you are parsing.  See
-    :file:`examples/load_demo.py`.
+    you know the date format string of the date you are parsing.
```

Comment:
@story645 I think quoting the first sentence would have clarified things; at first I thought it was just a statement instead of a suggestion of how to write the sentence.